olympia.py

```diff
@@ -16,13 +16,8 @@ def update_system_path():
     are importable without the `apps.` prefix."""
 
     ROOT = os.path.dirname(os.path.abspath(__file__))
-
-    prev_sys_path = set(sys.path)
-
-    site.addsitedir(os.path.join(ROOT, 'apps'))
-
-    # Move the new items to the front of sys.path.
-    sys.path.sort(key=prev_sys_path.__contains__)
+    # Insert the new items to the front of sys.path.
```

Comment:
I'd be more specific and say something like `# Insert the 'apps' folder to the front of sys.path so it takes precedence.`